src/test/java/picard/fingerprint/CrosscheckFingerprintsTest.java

```diff
@@ -256,9 +278,55 @@ public void testCrossCheckSMs(final File file1, final File file2, final int expe
         doTest(args, metrics, expectedRetVal, numberOfSamples * (numberOfSamples + 1) / 2, CrosscheckMetric.DataType.SAMPLE);
 
         TabbedTextFileWithHeaderParser matrixParser = new TabbedTextFileWithHeaderParser(matrix);
-        Assert.assertEquals(matrixParser.columnLabelsList().size(), numberOfSamples + 1 );
+        Assert.assertEquals(matrixParser.columnLabelsList().size(), numberOfSamples + 1);
+    }
+
+    @DataProvider(name = "checkSamplesData")
+    public Iterator<Object[]> checkSamplesData() {
+        List<Object[]> tests = new ArrayList<>();
```

Comment:
Need a test with an empty (or null) INPUT2.

Reply:
most of the cross-check tests are without INPUT2 ...this is just testing the new functionality